target_snowflake/__init__.py

```diff
@@ -170,6 +173,20 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
             else:
                 records_to_load[stream][primary_key_string] = o['record']
 
+            if archive_load_files_enabled and stream in archive_load_files_data:
+                # Keep track of min and max of the designated column
+                values = archive_load_files_data[stream]
+                archive_primary_column_name = values['column']
+                archive_primary_column_value = o['record'][archive_primary_column_name]
```

Comment:
What we should return if `archive_primary_column_name` is a composite primary key? How we should track it?